README.md

```diff
@@ -48,28 +52,45 @@ scrot requires a few projects and libraries:
 - libXfixes [(can be found in X.Org)](https://gitlab.freedesktop.org/xorg/lib/libxfixes)
 - libXinerama [(can be found in X.Org)](https://gitlab.freedesktop.org/xorg/lib/libxinerama)
 
-From a Unix command line, building is simple:
+### Generic installation instruction ###
+
+In short, if you are building from an GIT checkout of scrot (but not if you
+are building from a downloaded tarball) then you will first need to run:
 ```
-$ ./autogen.sh
-$ ./configure
-$ make
+./autogen.sh
 ```
 
-Cleaning up the build is also simple:
+TIP: if the source code does not have a configure file, you will need to run
+     the ```./autogen.sh``` command.
+
+Now, proceed as shown below for a tarball. (If you are using the tarball
+but have applied additional patches then you may also need to run
+```./autogen.sh```.)
+
+To build scrot from a downloaded tarball just type:
 ```
-$ make distclean
+./configure
+make
 ```
 
-You can return to a pristine source tree before running `./configure`:
+And as root:
 ```
-$ ./autogen.sh clean
+make install
 ```
 
-## Installing ##
+Cleaning up the build is also simple:
+```
+make clean
```

Comment:
Wait..
Missing the `$` character:`$ make clean` and `# make install`

Reply:
I removed because all other  previous commands was without the prompts. I prefer all commands with prompts. What you think?

Reply:
`$ ./autogen.sh` 
`$ ./configure`
 `$ make` 
`$ make distclean`
and others that must be out there. :)

Reply:
> I removed because all other previous commands was without the prompts. I prefer all commands with prompts. What you think?

Yes, I agree,  I also prefer it with prompts.